tox.ini

```diff
@@ -0,0 +1,20 @@
+# Linter https://github.com/wemake-services/wemake-python-styleguide
+# Settings set up https://wemake-python-stylegui.de/en/latest/pages/usage/configuration.html
+
+[flake8]
+max-methods = 25
+max-arguments = 8
+max-awaits = 8
+max-local-variables = 10
+max-cognitive-score = 15
+
+per-file-ignores =
+  #  WPS100 Found wrong module name
+  #  N805 first argument of a method should be named 'self'
+  #  WPS430 Found nested function
+  #  WPS420 Found wrong keyword: global
+  #  WPS442 Found outer scope names shadowing
+  #  WPS410 Found wrong metadata variable
+  #  WPS437 Found protected attribute usage:
+
+  */*.py: WPS100, N805, WPS430, WPS420, WPS442, WPS410, WPS437
```

Comment:
At first glance everything looks good and fits our current code base pretty well, i.e. we have a lot of `global`, `__author__` and outer scope names shadowing usage and while we're on our way to cover in tests as many features as possible, we can't really rewrite things constantly, so for now let's just have those "violations" detections disabled.
Have you already tried to apply this config to our tests? I haven't looked up closely [all](https://wemake-python-stylegui.de/en/latest/pages/usage/violations/index.html) possible warnings/errors since there are thousands of them and I just skimmed through, hence we can have this set as a baseline and work out final solution iteratively.

Reply:
Also `.pylint` is no use now, we can get rid of it.